qcodes/dataset/measurements.py

```diff
@@ -101,7 +104,7 @@ def add_result(self,
                 raise ValueError(f'Can not add a result for {paramstr}, no '
                                  'such parameter registered in this '
                                  'measurement.')
-            if isinstance(value, np.ndarray):
+            if sum([isinstance(value, typ) for typ in array_like_types]) > 0:
```

Comment:
Is the regular method of achieving this sort of comparison normally the `any` function?
```py3
any(isinstance(value, typ) for typ in array_like_types):
```

Reply:
Haha, it sure is. Thanks.